setup.py

```diff
@@ -16,7 +16,7 @@
     long_description = f.read()
 
 tests_require = [
-    'pytest-cov', 'hypothesis>=3.7.0', 'pytest>=3.4.0', 'py>=1.5.0',
+    'pytest-cov', 'hypothesis>=3.7.0', 'pytest==3.4.2', 'py>=1.5.0',
```

Comment:
Please only use `>=` requirements, it makes no sense to require an exact version and makes packaging on Nix more cumbersome.

Reply:
I am pushing the changes now.

Reply:
Thanks! I might have to patch it in nixpkgs though until the next version is released

Reply:
Understood. Sorry about that!

Reply:
@rachmadaniHaryono can you take a look on the Travis failure?

Reply:
OK, please raise the PR and approve yourself ;).

Reply:
@Infinisil could you please consider submitting [pdd](https://github.com/jarun/pdd) to Nix?

Reply:
@jarun https://github.com/NixOS/nixpkgs/pull/38395 :)

Reply:
Thank you!